ccswbs/gus
# Slate Forms Styles

# Summary of changes
Slate forms do not use Bootstrap classes. To make the form appear with Bootstrap styles, this branch adds slate form styles to the Content Hub

## Frontend
Adding slate form styles to the Content Hub.

## Backend
None

# Test Plan

- View page with embedded Slate form
- Ensure page appears with Bootstrap-ish styles
